GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.Designer.cs

```diff
@@ -457,7 +457,7 @@ private void InitializeComponent()
             0});
             this._NO_TRANSLATE_DaysToCacheImages.Name = "_NO_TRANSLATE_DaysToCacheImages";
             this._NO_TRANSLATE_DaysToCacheImages.Size = new System.Drawing.Size(38, 20);
-            this._NO_TRANSLATE_DaysToCacheImages.TabIndex = 3;
+            this._NO_TRANSLATE_DaysToCacheImages.TabIndex = 2;
```

Comment:
Avatar provider should be after this but is two tabs later

Reply:
Fix coming up.